fic/eri/v1/router_paired_to_gcp_connections/requests.go

```diff
@@ -76,22 +87,24 @@ type Destination struct {
 	// AWSAccountID string            `json:"awsAccountId" required:"true"`
```

Comment:
Could you please remove these unwanted comments throughout?

Reply:
DONE